src/mqt/predictor/driver.py

```diff
@@ -5,10 +5,14 @@ def compile(qc, model="ML", opt_objective="fidelity"):
     if model == "ML":
         predictor = ml.Predictor()
         prediction = predictor.predict(qc)
-        return predictor.compile_predicted_compilation_path(qc, prediction)
+        compiled_qc = predictor.compile_predicted_compilation_path(qc, prediction)
+        compile_information = ml.helper.get_index_to_comppath_LUT()[prediction]
+        return compiled_qc, compile_information
+
     elif model == "RL":
         predictor = rl.Predictor()
-        return predictor.compile(qc, opt_objective=opt_objective)
+        compiled_qc, compile_information = predictor.compile(qc, opt_objective)
+        return compiled_qc, compile_information
```

Comment:
The change here hasn't really accomplished anything 🙃

Reply:
Implemented now in new structure.